doc/nrf/known_issues.rst

```diff
@@ -1008,6 +1008,13 @@ KRKNWK-6073: Potential delay during FOTA
 Matter
 ======
 
+.. rst-class:: v2-3-0 v2-2-0
+
+KRKNWK-16575: Applications with factory data support do not boot up properly on nRF5340
+  When the Matter sample is built for ``nrf5340dk_nrf5340_cpuapp`` target with :kconfig:option:`CONFIG_CHIP_FACTORY_DATA` set to 'y' the application returns prematurely with error code 200016 because the factory data partition is not aligned to the :kconfig:option:`CONFIG_FPROTECT_BLOCK_SIZE`.
```

Comment:
```suggestion
  When the Matter sample is built for ``nrf5340dk_nrf5340_cpuapp`` build target with the :kconfig:option:`CONFIG_CHIP_FACTORY_DATA` Kconfig option set to ``y``, the application returns prematurely the error code 200016 because the factory data partition is not aligned with the :kconfig:option:`CONFIG_FPROTECT_BLOCK_SIZE` Kconfig option.
```